README.md

```diff
@@ -283,7 +283,10 @@ And finally, you are ready to go! Or at least you should be...
 
 ### Junior Contributors
 
+**Lucas Silva** | [brachingo] | I love driving, I have an Opel Corsa Sport
+and a Kawasaki, and driving them makes me feel free
```

Comment:
Almost!

In order to do a propper hyperlink you must place next to your name, again your name (in this concrete case) in order to make a link to the line where you have inserted the link.